lib/celluloid/calls.rb

```diff
@@ -132,7 +132,6 @@ def wait
 
   # Asynchronous calls don't wait for a response
   class AsyncCall < Call
-
```

Comment:
These are the types of code compressions I am talking about, in a few internal Celluloid classes.